docs/content/en/docs/install/install.md

```diff
@@ -102,30 +126,47 @@ modify some values, and use the modified file to install Keptn:
 
 1. Edit your local copy to modify some values
 
-1. Install Keptn by adding the following string to your `helm upgrade` command line:
+1. To apply the modified configuration.
+   install Keptn by adding the following string
+   to your `helm upgrade` command line:
```

Comment:
i don't think this additional text is needed since we only talk about modifying the config here anyways. so we don't need to emphasise again, that we want to modify the config

Reply:
done